CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Ansible Collections for HPE OneView Change Log
+
+## v6.4.0 (unreleased)
+#### Bug fixes & Enhancements
+- [#131] (https://github.com/HewlettPackard/oneview-ansible-collection/issues/131) Gather facts about a ID Pools IPV4 Subnet by name doesnt supported.
```

Comment:
I think you forgot to link 139 here. Issue 131 was fixed and merge in 6.3.0 release.

Reply:
done